app/controllers/poll_skips_controller.rb

```diff
@@ -2,12 +2,15 @@ class PollSkipsController < ApplicationController
   before_action :authenticate_user!, only: %i[create]
 
   def create
-    @poll_skip = PollSkip.find_or_create_by(poll_id: poll_skips_params[:poll_id], user_id: current_user.id)
-    @poll = Poll.find(poll_skips_params[:poll_id])
-    render json: { voting_data: @poll.voting_data,
-                   poll_id: poll_skips_params[:poll_id].to_i,
-                   user_vote_poll_option_id: nil,
-                   voted: false }
+    poll = Poll.find(poll_skips_params[:poll_id])
+    poll.poll_skips.create_or_find_by(user_id: current_user.id)
```

Comment:
> The primary key may auto-increment on each create, even if it fails.

Thats an interesting tidbit to keep in mind if we decide to use this more.

Reply:
@mstruve ahaha we're back to this issue https://github.com/thepracticaldev/tech-private/issues/385 and this PR https://github.com/thepracticaldev/dev.to/pull/1113